### DISCUSSION TITLE
Updates are not working

### DISCUSSION THREAD

**Discussion by Jojo1703** [mworion/MountWizzard4#366] [Q&A]
Hello,
I have the problem that no more updates are being sent to the mount (GM3000). Satellites, comets, asteroids are not working. The earth rotation data cannot be updated either, see attachement. Version MW4 is the latest 3.2.7b0, mount is 3.1.10.
Everything was working properly a few months ago.
Is there a solution?
Regards
Jurgen
![leap](https://github.com/user-attachments/assets/93f8d360-f161-4db2-a058-6b4523d36bd3)
![Satupdate](https://github.com/user-attachments/assets/24214606-29ac-4cf8-b65d-788ee3fa5148)

**Comment by mworion** [comment]
Hi Jurgen,
sorry to hear. I countercheck this, no issue in my setup, could you please zip and post the CDFLeapSeconds.txt out of your work dir under /data ? 
And please post the log as well here to the webpage.
Did you update the firmware in between ? 
We'll find out about the issue.
Michel

**Comment by mworion** [comment]
I made an issue out of this discussion, please follow here:

https://github.com/mworion/MountWizzard4/issues/367

Michel
